pkg/localkube/proxy.go

```diff
@@ -38,6 +40,11 @@ func (lk LocalkubeServer) NewProxyServer() Server {
 }
 
 func StartProxyServer(lk LocalkubeServer) func() error {
+	fmt.Println(lk.APIServerAddress)
+	bindaddress := lk.APIServerAddress.String()
+	if lk.APIServerInsecurePort != 0 {
+		bindaddress = lk.APIServerInsecureAddress.String()
```

Comment:
Might be worth it to split this out into another helper function where the logic is clearer:

If insecure serving and and secure serving are both enabled, the insecure serving address will be used as the bind address.

like

```golang
func (LocalkubeServer) GetAPIServerAddress() net.IP {
  ...
}
```

which you could use here, anywhere else that needs to know how to talk to the API server, and for the healthchecks